.travis.yml

```diff
@@ -1,16 +1,14 @@
 dist: xenial
 language: python
 python:
- - "3.4"
- - "3.5"
```

Comment:
Is there a reason to stop supporting these versions?

Reply:
There were some issues with the code + Python 3.5 stopped with full support in mid 2017 iirc, and ended security fixes 2 years later... The earliest Python version still supported is 3.6 (till end of next year).
If you want I will add them again?

Reply:
3.5 or 3.4? 3.5 end of life was [2 weeks ago](https://www.python.org/downloads/release/python-3510/) (it has now entered in security mode only)... But yep, still, it is in the past.

I spend a bit of time on my last PR because of some python<3.6 related issues (unordered dict).

There are some stats on the marketshare of python versions here: https://github.com/hugovk/pypi-tools/tree/master/ and... I think I am in favor of dropping support for 3.5 in the new versions of pysparkling. Its marketshare is shrinking (more than I thought), there is a cost to maintain them, and the previous version of pysparkling still supports it